.github/workflows/build.yaml

```
@@ -0,0 +1,64 @@
+name: Build
+
+on:
+    push: ~
+    pull_request: ~
+    release:
+        types: [created]
+    schedule:
+        -
+            cron: "0 1 * * 6" # Run at 1am every Saturday
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+
+        name: "PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}"
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [7.4, 8.0]
```

Can we check 8.1 as well?
```suggestion
                php: [7.4, 8.0, 8.1]
```